examples/deepspeed/gpt_neox/gpt2_trial.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, context: DeepSpeedTrialContext) -> None:
         except:
             traceback.print_exc()
             raise InvalidHP("Could not parse neox_args.")
+        print(self.neox_args)
```

Comment:
nit:  should we remove this print statement? is it important to keep it? If so, maybe let's use logging.

Reply:
good point, will change to logging